app/Resources/views/composer/step2.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends "composer/step_layout.html.twig" %}
+
+{% block step_title %}
+    Step 2: Install the database
+{% endblock %}
+
+{% block step_description %}
+    You need a MySQL server to store the database.<br/>
+    We are going to create the Database and import the schema and the first clean dataset.
+{% endblock %}
+
+
+{% block step_contnet %}
+    <div class="help">
+        <p>If you do not have a MySQL server ready, you can create one quickly using Docker.</p>
+    </div>
+    <div class="orange">
+        <p>&#36; docker run --rm -p 3333:3306 --name ezdbcontainer -e MYSQL_ROOT_PASSWORD=ezplatform mariadb:5.5</p>
```

Comment:
Also we would need instructions for installing docker.
On mac I'm unsure if this is easier than just linking to mysql 5.6 mac download for instance and it _might_ be faster, but other hand then we need to start to describe install per platform.